index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>My test page</title>
+    <link href="styles/style.css" rel="stylesheet" type="text/css">
```

Comment:
Hey @Rrroberrrto, this file does not exist in the repo. This will create confusion for other learners, so I would not recommend adding it here.